src/Hotkeys.ts

```diff
@@ -33,7 +34,17 @@ document.addEventListener('keydown', event => {
         return event.stopPropagation();
     }
 
-    const key = event.key.toUpperCase();
+    let keyPrefix = '';
+    if(event.ctrlKey) {
+        keyPrefix += 'CTRL+';
+    }
+    if(event.shiftKey) {
+        keyPrefix += 'SHIFT+';
+    }
+    if(event.altKey) {
+        keyPrefix += 'ALT+';
+    }
```

Comment:
```suggestion
    if (event.ctrlKey) {
        keyPrefix += 'CTRL+';
    } else if (event.shiftKey) {
        keyPrefix += 'SHIFT+';
    } else if (event.altKey) {
        keyPrefix += 'ALT+';
    }
```

Reply:
I wanted to let the user combine them if they wanted, e.g. `CTRL+ALT+<key>`, in that order

Reply:
alright, then all that needs changing is the space between the if and parenthesis

Reply:
Done 👍